src/openrct2/ride/ShopItem.cpp

```diff
@@ -13,70 +13,85 @@
 #include "../localisation/StringIds.h"
 #include "../sprites.h"
 
+ShopItem& operator++(ShopItem& d, int)
```

Comment:
These shouldn't be required. Try to avoid requiring them.

Reply:
I'll check tomorrow where I used it and delete if possible.